src/components/step-flow/step-flow.style.ts

```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+import Typography from "../typography/typography.component";
+
+const StyledStepFlow = styled.div``;
+
+const StyledStepContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledStepContentText = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledCategoryText = styled(Typography)`
+  color: var(--colorsUtilityYin055);
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 21px;
+  margin: 0px;
+  padding: 0px;
+`;
+
+const StyledTitleText = styled(Typography)``;
+
+const StyledTitleTextContainer = styled.span``;
+
+const StyledVisibleTitleText = styled.span`
```

Comment:
Comment: if you simplify the HTML like Ive suggested above you can probably just pass these to `Typography` directly